Python/scoreP3.py

```diff
@@ -93,21 +98,22 @@ def scoreP3(truthDir, testDir, phaseNum='3'):
     # Load the ground truth CSV file, and merge it with the test results
     combinedRows = []
     with open(truthFile) as truthFileObj:
-        truthReader = csv.DictReader(truthFileObj,
-                                     fieldnames=['image', 'confidence'])
+        truthReader = csv.DictReader(truthFileObj)
         for truthRow in truthReader:
             # Find the matching test result
-            testRow = matchRowName(truthRow['image'], testRows)
+            testRow = matchRowName(truthRow['image_id'], testRows)
 
             combinedRows.append({
-                'image': truthRow['image'],
-                'truth_value': float(truthRow['confidence']),
-                'test_value': testRow['confidence'],
+                'image_id': truthRow['image_id'],
+                'truth_value_mel': float(truthRow['melanoma']),
+                'test_value_mel': testRow['melanoma'],
+                'truth_value_sebk': float(truthRow['seborrheic_keratosis']),
+                'test_value_sebk': testRow['seborrheic_keratosis'],
             })
 
     # Build the Numpy arrays for calculations
```

Comment:
WIP: All of the metrics below this point must be computed for both `melanoma` and `seborrheic_keratosis`, along with a combined metric for both.